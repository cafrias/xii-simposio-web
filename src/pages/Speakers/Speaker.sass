@import '_sass/_constants.sass'
@import '_sass/_breakpoints.sass'

.speaker
  width: 100%
  text-align: center
  margin-bottom: 2.618rem

.speaker__pic
  border-radius: 50%
  max-width: 180px
  +shadow-1

.speaker__name
  font-weight: 900
  color: #fff
  font-size: 1.618rem
  margin-bottom: 0
  +anti-aliased

.speaker__inst
  color: $light-brown
  margin: 0

+for-tablet-portrait-up
  .speaker
    width: 48%

+for-desktop-up
  .speaker
    width: 33%
