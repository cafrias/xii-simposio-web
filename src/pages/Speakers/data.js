// @flow
import pic01 from './assets/01.jpg'
import pic02 from './assets/02.jpg'
import pic04 from './assets/04.jpg'
import pic05 from './assets/05.jpg'
import pic06 from './assets/06.jpg'
import pic07 from './assets/07.jpg'
import pic08 from './assets/08.jpg'

export default [
  {
    id: '01',
    pic: pic01,
    name: 'Ricardo Pahlen Acuña',
    inst: '(UBA)'
  },
  {
    id: '02',
    pic: pic02,
    name: 'Ana María Campo',
    inst: '(UBA)'
  },
  {
    id: '04',
    pic: pic04,
    name: 'Ricardo Barrera',
    inst: '(UNTDF-UNPSJB)'
  },
  {
    id: '05',
    pic: pic05,
    name: 'Francisco Provenzani',
    inst: '(UBA)'
  },
  {
    id: '06',
    pic: pic06,
    name: 'Hugo Bottino',
    inst: '(UBA)'
  },
  {
    id: '07',
    pic: pic07,
    name: 'Juan Permuy',
    inst: '(UBA)'
  },
  {
    id: '08',
    pic: pic08,
    name: 'Mauricio Lambertucci',
    inst: '(UBA)'
  }
]
