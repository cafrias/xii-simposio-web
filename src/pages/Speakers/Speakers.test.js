import React from 'react'
import { mount, shallow } from 'enzyme'

import Speakers from './index'

it('should render without errors', () => {
  const wrapper = mount(<Speakers/>)
  expect(wrapper).toHaveLength(1)
})

it('should match snapshot', () => {
  const wrapper = shallow(<Speakers/>)
  expect(wrapper).toMatchSnapshot()
})
