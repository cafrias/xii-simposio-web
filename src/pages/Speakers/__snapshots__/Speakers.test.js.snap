// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<Section
  className="background_orange speakers"
  id="speakers"
  name="Disertantes"
>
  <ul
    className="speakers__grid"
  >
    <Speaker
      id="02"
      inst="(UBA)"
      key="0"
      name="Ana María Campo"
      pic="02.jpg"
    />
    <Speaker
      id="04"
      inst="(UNTDF-UNPSJB)"
      key="1"
      name="Ricardo Barrera"
      pic="04.jpg"
    />
    <Speaker
      id="05"
      inst="(UBA)"
      key="2"
      name="Francisco Provenzani"
      pic="05.jpg"
    />
    <Speaker
      id="07"
      inst="(UBA)"
      key="3"
      name="Juan Permuy"
      pic="07.jpg"
    />
    <Speaker
      id="08"
      inst="(UBA)"
      key="4"
      name="Mauricio Lambertucci"
      pic="08.jpg"
    />
  </ul>
</Section>
`;
